client/my-sites/email/email-management/home/email-plan-mailboxes/list-header.tsx

```diff
@@ -51,30 +51,32 @@ const MailboxListHeader = ( {
 
 	return (
 		<div className="email-plan-mailboxes-list__mailbox-list">
-			<SectionHeader
-				isPlaceholder={ isPlaceholder }
-				label={
-					<>
-						{ !! showIcon && domain && <HeaderIcon /> }
-						{ getListHeaderTextForAccountType() }
-					</>
-				}
-			>
-				{ addMailboxPath && (
-					// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-					// @ts-expect-error
-					<Button href={ addMailboxPath } variant="link" disabled={ !! disableActions }>
-						{ translate( 'Add mailbox' ) }
-					</Button>
-				) }
-				{ showContextMenu && (
-					<ContextMenu
-						domain={ domain }
-						className="email-plan-mailboxes-list__mailbox-context-menu"
-					/>
-				) }
-			</SectionHeader>
-			{ children }
+			<div className="vertical-nav">
+				<SectionHeader
+					isPlaceholder={ isPlaceholder }
+					label={
+						<>
+							{ !! showIcon && domain && <HeaderIcon /> }
+							{ getListHeaderTextForAccountType() }
+						</>
+					}
+				>
+					{ addMailboxPath && (
+						// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+						// @ts-expect-error
+						<Button href={ addMailboxPath } variant="link" disabled={ !! disableActions }>
+							{ translate( 'Add mailbox' ) }
+						</Button>
+					) }
+					{ showContextMenu && (
+						<ContextMenu
+							domain={ domain }
+							className="email-plan-mailboxes-list__mailbox-context-menu"
+						/>
+					) }
+				</SectionHeader>
+				{ children }
+			</div>
```

Comment:
```suggestion
			<SectionHeader
				isPlaceholder={ isPlaceholder }
				label={
					<>
						{ !! showIcon && domain && <HeaderIcon /> }
						{ getListHeaderTextForAccountType() }
					</>
				}
			>
				{ addMailboxPath && (
					// eslint-disable-next-line @typescript-eslint/ban-ts-comment
					<Button href={ addMailboxPath } variant="link" disabled={ !! disableActions }>
						{ translate( 'Add mailbox' ) }
					</Button>
				) }
				{ showContextMenu && (
					<ContextMenu
						domain={ domain }
						className="email-plan-mailboxes-list__mailbox-context-menu"
					/>
				) }
			</SectionHeader>
			{ children }
```
It looks like the only purpose `vertical-nav` is to add a `margin-top: 15px` rule in the single-site context. Applying that style to `email-plan-mailboxes-list__mailbox-list` directly seems sensible, since we're not using `VerticalNavItem` here.

Side note: the `// @ts-expect-error` looks to be superfluous unless I'm missing something.

Reply:
Thanks for the quick review, Fredrik. All makes sense! I've moved the styling to the component and cleaned up the `vertical-nav` (not sure why I went with the div instead of the component). Can you give it another quick look? There should be no visual changes.

Regarding the `@ts-expect-error`, I've tried removing it but that unveiled a strange TypeScript error related to the `href`. 🤷 I wasn't able to find a reasonable explanation, so I'm leaving it for now.